load.py

```diff
@@ -202,39 +206,46 @@ def plugin_stop():
 def journal_entry(cmdr, is_beta, system, station, entry, state):
     global planet
     global landingPad
+    global ship
+    if showShipName:
+        #create ship name text if enabled.
+        shipt = " with " + state["ShipName"]
```

Comment:
Move the `with` in the translation file since that word will vary from language to language

Reply:
Ok, will do. 
About the spacing, the `{shipname}` param will be in the end of the string with no spacing (the space is inside it) so when disabled there's no extra space at the end.